inspections/codesniffer/phpcs.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+function remove_diff_range {
+  sed 's/:[0-9][0-9]*-[0-9][0-9]*$//' | sort | uniq
+}
+
+function lint_php_files {
+	PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+	TEMP_DIRECTORY=$(mktemp -d 2>/dev/null || mktemp -d -t 'dev-lib')
+
+	DIFF_BASE=${DIFF_BASE:-origin/master}
```

Comment:
```suggestion
	DIFF_BASE="origin/${DIFF_BASE:-master}"
```

This way, the instructions in the README can become much simpler. The user can only specify `master` instead of `origin/master`